torch/fx/graph_module.py

```diff
@@ -37,11 +37,14 @@ def patched_getline(*args, **kwargs):
     return _orig_getlines(*args, **kwargs)
 linecache.getlines = patched_getline
 
+
 def _forward_from_src(src: str, globals: Dict[str, Any]):
     # avoid mutating the passed in dict
     globals = globals.copy()
```

Comment:
I would prefer that this be named something like `d` or `globals_copy` instead. I had to think for a second about Python scoping rules, pass-by semantics, and whether the dict `copy` function makes a shallow vs. deep copy to ensure that this line was correct

Reply:
Changed it to `globals_copy`.